src/Gettext.Tools/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Diagnostics;
+using System.Reflection;
+
+namespace GetTextToolsWindows.DotNetTool
+{
+    public class Program
+    {
+        public static int Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                ShowHelp();
+
+                return 1;
+            }
+
+            var binFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!;
+            var gettextBinFolder = Path.Combine(binFolder, "..", "..", "bin");
+
+            var toolName = args[0];
+            var toolFile = Path.Combine(gettextBinFolder, $"{toolName}.exe");
+
+            if (!File.Exists(toolFile))
+            {
+                Console.WriteLine("Invalid gettext tool.");
+                return 2;
+            }
+
+
+            var startInfo = new ProcessStartInfo(toolFile)
+            {
+                Arguments = string.Join(" ", args.Skip(1)),
+                UseShellExecute = false,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                CreateNoWindow = false
+            };
+
+            // needed to correctly print output from gettext tools
+            Console.OutputEncoding = System.Text.Encoding.Latin1;
+
+            using (var p = Process.Start(startInfo))
+            {
+                if (p == null) return 3;
+
+
+                Console.WriteLine(p.StandardOutput.ReadToEnd());
+                Console.WriteLine(p.StandardError.ReadToEnd());
+
+                p.WaitForExit(10000);
+
+                return p.ExitCode;
+            }
+        }
+
+        private static void ShowHelp()
+        {
+            Console.WriteLine("Usage:");
+            Console.WriteLine("  Gettext.Tools -- <TOOL_NAME> [tool specific options]");
+            Console.WriteLine();
```

Comment:
I don't really understand how to use it from this help. It would be good to add some documentation to the readme file too...

Reply:
I have added more information to the usage output and additionally added a readme.md file for the dotnet tool nuget package.